xapian-core/include/xapian/termgenerator.h

```diff
@@ -141,6 +144,17 @@ class XAPIAN_VISIBILITY_DEFAULT TermGenerator {
      */
     void set_stemming_strategy(stem_strategy strategy);
 
+    /** Set the stopper strategy.
+     *
+     *  @param strategy The strategy to use - possible values are:
+     *   - STOP_NONE:   Disable stopper for both the stemmed and unstemmed form 
+     *           of the word (Don't use the stopper).  
+     *   - STOP_ALL:    Don't disable stopper for both forms of the word. 
+     *   - STOP_STEMMED:  Disable stopper for the unstemmed form of the
+     *           word.   
```

Comment:
It looks slightly neater if you align the second line with the description (likewise for `STOP_NONE`). (Look at the `STEM_` documentation above.) Also, rephrasing as positive as above would be helpful.